packages/core/app-layout/src/components/aboutSection/AppAboutSection.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+  <KCard class="kong-ui-app-about-section">
+    <template
+      v-if="title"
+      #title
+    >
+      <div class="about-section-title">
+        {{ title }}
+      </div>
+    </template>
+
+    <template
+      v-if="$slots.actions || created || modified"
+      #actions
+    >
+      <!-- TODO: translation -->
+      <div class="about-section-header-end">
+        <div
+          v-if="created || modified"
+          class="about-section-timestamps"
+          :class="{ 'has-actions': $slots.actions }"
+        >
+          <span
+            v-if="created"
+            class="about-section-timestamps-created"
+          >{{ createdLabel || 'Created' }}: {{ created }}</span>
+          <!-- TODO: use new icon instead of KIcon -->
+          <span
+            v-if="created && modified"
+            class="about-section-timestamps-arrow"
+          >-></span>
+          <span
+            v-if="modified"
+            class="about-section-timestamps-modified"
+          >{{ modifiedLabel || 'Modified' }}: {{ modified }}</span>
+        </div>
+
+        <div
+          v-if="$slots.actions"
+          class="about-section-actions"
+          data-testid="about-section-actions"
+        >
+          <slot name="actions" />
+        </div>
+      </div>
+    </template>
+
+    <template #body>
+      <div
+        v-if="description"
+        class="about-section-description"
+      >
+        {{ description }}
+      </div>
+
+      <div
+        v-if="$slots.default"
+        class="about-section-content"
+      >
+        <slot />
+      </div>
+
+      <hr v-if="$slots['divider-section']">
+
+      <div
+        v-if="$slots['divider-section']"
+        class="about-divider-section"
+      >
+        <slot name="divider-section" />
+      </div>
+    </template>
+  </KCard>
+</template>
+
+<script setup lang="ts">
+defineProps({
+  title: {
+    type: String,
+    default: '',
+  },
+  description: {
+    type: String,
+    default: '',
+  },
+  created: {
+    type: String,
+    default: '',
+  },
+  createdLabel: {
+    type: String,
+    default: '',
+  },
+  modified: {
+    type: String,
+    default: '',
+  },
+  modifiedLabel: {
+    type: String,
+    default: '',
+  },
```

Comment:
question: why would the `createdLabel` or `modifiedLabel` ever be different text?

Reply:
If that's the case, should the prop names for `created/modified` be more generic?

e.g. `createdLabel: 'Joined'` seems a bit confusing

Reply:
Do you have a suggestion, it maps to `created_at`/`updated_at` fields most of the time.